scania-digital-design-system/sdds
# docs(tabs): removes the folder structure for the tabs stories

**Describe pull-request**  
Removes the folder structure for the tabs component.

**Solving issue**  
Fixes: [AB#2687](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2687)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Tabs
3. Check the stories.

**Screenshots**  
-

**Additional context**  
-
